model/src/main/java/org/mobilitydata/gtfsvalidator/notice/SeverityLevel.java

```diff
@@ -38,5 +38,18 @@ public enum SeverityLevel {
    * (e.g., using the language "must"). The validator uses RFC2119
    * (https://tools.ietf.org/html/rfc2119) to interpret the language in the GTFS spec.
    */
-  ERROR
+  ERROR;
+
+  public static int getOrder(SeverityLevel severityLevel) {
```

Comment:
Is it not more typical to put values in the enum elements themselves and just return that value?

Reply:
Yes. As David pointed out it was much easier to just use the enum's reverse order.